src/assets/PythonCopyFileWithShutil/python-copy-paste.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Ways to copy file with shutil module in Python
+authorName: Salipa Gurung
+authorAvatar: https://avatars.githubusercontent.com/u/53458341?v=4
+authorLink: https://github.com/Salipa-Gurung
+createdAt: Feb 27, 2024
+tags: python, copy-paste, shutil
+banner: https://blog.jankaritech.com/src/assets/PythonCopyFileWithShutil/images/python_copy_file.png
+---
+
+Python provides many built-in modules to copy and paste a file. We'll look into ways to copy file using `shutil` module in this blog post.
+
+The standard utility module known as shutil is an abbreviation for shell utilities. Numerous functions in the Python shutil module helps with high-level file and directory operations. Using the shutil module, there are four common methods that provide an easy and efficient way to copy and paste a file:
+
+### 1. shutil.copy() method
+
+```py
+shutil.copy(source, destination, *, follow_symlinks=True)
+```
+
+This method copies the file of source to the directory or file specified in the destination and returns file's destination.
+
+- shutil.copy() method copies file's data and file's permission mode.
+- Metadata(file creation and modification time) of the file copied is not preserved.
+- Source and destination should be path-like objects or strings.
+- Destination can specify both file or directory. If the destination is directory, the file will be copied with the same name as of the source file name.
+- Copying symbolic link:
```

Comment:
A list should have some kind of "heading" that indicates what the list is about.

Food I love:
- momo
- biryani
- ...